package/opt/hassbian/suites/install_duckdns.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+function duckdns-show-short-info {
+echo -e "Setup for duckdns auto renewal"
+}
+
+function duckdns-show-long-info {
+echo -e "This script adds an cron job to uppdate you duckdns IP address."
```

Comment:
It doesn't update the IP address ;)

Reply:
Not sure how else to put it :(

Updated description to this:
This script adds an cron job to auto uppdate you the WAN IP address for the defined domain.